data/private

```diff
@@ -5,9 +5,19 @@ local
 localhost
 
 # Also defined in rfc2606
-# test
-# example
-# invalid
+test
+example
+invalid
+
+# References:
+# https://www.iana.org/assignments/special-use-domain-names/special-use-domain-names.xhtml
+# https://unix.stackexchange.com/questions/92441/whats-the-difference-between-local-home-and-lan
+localdomain
+lan
+home.arpa
+
+# Dotless domains
+regexp:^[^.]+$
 
 # OpenWrt LuCI Web Interface
 openwrt.lan
```

Comment:
Since we have `lan` now, `openwrt.lan` can be removed.

Reply:
https://github.com/v2fly/domain-list-community/pull/230